default/scripting/policies/CONFEDERATION.focs.txt

```diff
@@ -18,20 +18,20 @@ Policy
             ]
             priority = [[DEFAULT_PRIORITY]]
             effects = 
-                SetTargetInfluence value = value + 1
+                SetTargetInfluence value = value + (NamedReal name = "PLC_CONFEDERATION_MAXINFLUENCE_EFFECT" value = 1)
 
         // makes all planets less stable
         EffectsGroup
-        scope = And [
-            Planet
-            OwnedBy empire = Source.Owner
-            Population low = 0.001
-        ]
-        priority = [[DEFAULT_PRIORITY]]
-        effects = [
-            SetTargetHappiness value = Value - 1
-            SetMaxTroops value = Value + 5
-        ]
+            scope = And [
+                Planet
+                OwnedBy empire = Source.Owner
+                Population low = 0.001
+            ]
+            priority = [[VERY_EARLY_PRIORITY]]
+            effects = [
+                SetTargetHappiness value = Value - (NamedReal name = "PLC_CONFEDERATION_MAXHAPPINESS_EFFECT" value = 1)
+                SetMaxTroops value = Value + (NamedReal name = "PLC_CONFEDERATION_MAXTROOPS_EFFECT" value = 5)
```

Comment:
could be named _BONUS instead of _EFFECT

Reply:
Policy effects are still WIP, some could change quite a bit, even by changing a malus for a bonus or vice versa, so I wouldn't use BONUS or MALUS.
Maybe we can find a better word than effect. VALUE?
Also, the negative sign could be included in the NameReal value and always use **+** before NameReal/Integer:
`
value = Value + (NamedReal name = "PLC_WHATEVER_VALUE" value = (-1))
`

Reply:
_VALUE is not providing any context. Use _BONUS now, and lets rename it when we change the operator from + to *,- to _FACTOR,_MALUS accordingly.

Reply:
I opted for "_FLAT" and "_PERPOP".
BONUS does not tell you if it is per pop or flat. FACTOR does not tell you if it is a per pop addition or a value scaling. I would use SCALING for the latter (not relevant for this PR).